src/net.cpp

```diff
@@ -2202,9 +2206,9 @@ std::vector<AddedNodeInfo> CConnman::GetAddedNodeInfo() const
         }
     }
 
-    for (const std::string& strAddNode : lAddresses) {
-        CService service(LookupNumeric(strAddNode, Params().GetDefaultPort(strAddNode)));
-        AddedNodeInfo addedNode{strAddNode, CService(), false, false};
+    for (auto nodes_to_add : lAddresses) {
```

Comment:
```suggestion
    for (auto node_to_add : lAddresses) {
```

(since you're iterating over the items individually, `node_to_add` is singular)

Reply:
done